src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -1477,6 +1478,83 @@ class MavlinkStreamCameraTrigger : public MavlinkStream
 	}
 };
 
+class MavlinkStreamCameraImageCaptured : public MavlinkStream
+{
+public:
+	const char *get_name() const
+	{
+		return MavlinkStreamCameraImageCaptured::get_name_static();
+	}
+
+	static const char *get_name_static()
+	{
+		return "CAMERA_IMAGE_CAPTURED";
+	}
+
+	static uint16_t get_id_static()
+	{
+		return MAVLINK_MSG_ID_CAMERA_IMAGE_CAPTURED;
+	}
+
+	uint16_t get_id()
+	{
+		return get_id_static();
+	}
+
+	bool const_rate()
+	{
+		return true;
+	}
+
+	static MavlinkStream *new_instance(Mavlink *mavlink)
+	{
+		return new MavlinkStreamCameraImageCaptured(mavlink);
+	}
+
+	unsigned get_size()
+	{
+		return (_image_time > 0) ? MAVLINK_MSG_ID_CAMERA_IMAGE_CAPTURED_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES : 0;
+	}
+
+private:
+	MavlinkOrbSubscription *_trigger_sub;
+	uint64_t _image_time;
+
+	/* do not allow top copying this class */
+	MavlinkStreamCameraImageCaptured(MavlinkStreamCameraImageCaptured &);
+	MavlinkStreamCameraImageCaptured &operator = (const MavlinkStreamCameraImageCaptured &);
+
+protected:
+	explicit MavlinkStreamCameraImageCaptured(Mavlink *mavlink) : MavlinkStream(mavlink),
+		_trigger_sub(_mavlink->add_orb_subscription(ORB_ID(camera_image))),
+		_image_time(0)
+	{}
+
+	void send(const hrt_abstime t)
+	{
+		struct camera_image_s image;
+
+		if (_trigger_sub->update(&_image_time, &image)) {
+
+			mavlink_camera_image_captured_t msg;
+
+			msg.time_boot_ms = image.timestamp / 1000;
+			msg.time_utc = image.timestamp_utc;
+			msg.camera_id = 1;
```

Comment:
This should probably be 0 for all camera IDs, unless the trigger is specifically sent to camera ID 1 only.